docs/community/release-checklist.md

```diff
@@ -0,0 +1,340 @@
+be spent regression testing by testing new features and upgrade paths from previous releases.
+
+During this phase, `$RELEASE_CANDIDATE_NUMBER` will keep incrementing as you produce new release
+candidates. The frequency of new candidates is up to the release manager: use your best judgement
+taking into account the severity of reported issues, testers' availability, and the release deadline
+date. Generally speaking, it is better to delay a release a few days than to ship something that is
+broken.
+
+Each time you'll want to produce a new release candidate, you will start by adding commits to the
+branch by cherry-picking from `master`:
+
+```console
+git cherry-pick -x <commit_id>
+```
+
+After that, tag it and notify users of the new release candidate:
+
+```console
+export RELEASE_CANDIDATE_NUMBER="2"
+git tag --sign --annotate "${RELEASE_CANDIDATE_NAME}" --message "Krustlet release ${RELEASE_CANDIDATE_NAME}"
+git push upstream $RELEASE_CANDIDATE_NAME
+```
+
+From here on, repeat the process until you're happy with the release candidate.
+
+When you're finally happy with the quality of a release candidate, you can move on and create the
+tag. Double-check one last time to make sure everything is in order, then finally push the release
+tag.
+
+```console
+git checkout $RELEASE_BRANCH_NAME
+git tag --sign --annotate "${RELEASE_NAME}" --message "Krustlet release ${RELEASE_NAME}"
+git push upstream $RELEASE_NAME
+```
+
+It is usually more beneficial to the end-user if the release notes are hand-written by a human
+being/marketing team/dog, so we'll go ahead and write up the release notes.
+
+If you're releasing a major/minor release, listing notable user-facing features is usually
+sufficient. For patch releases, do the same, but make note of the symptoms causing the original
+issue, who may be affected, and how the patch mitigates the issue.
+
+it should look like this:
+
+```markdown
+## Krustlet vX.Y.Z
+
+Krustlet vX.Y.Z is a feature release. This release, we focused on <insert focal point here>. Users are encouraged to upgrade for the best experience.
+
+## Notable features/changes
+
+- TLS bootstrapping support has been added
+- Improved error handling for the Kubelet API
+
+## Breaking changes
+
+No known breaking changes were introduced this release.
+
+## Known issues/missing features
+
+- Needs more cowbell
+- I gotta have more cowbell!
+
+## Installation
+
+Download Krustlet vX.Y.Z:
+
+- [checksums-vX.Y.Z.txt](https://krustlet.blob.core.windows.net/releases/checksums-vX.Y.Z.txt)
+- [krustlet-vX.Y.Z-linux-amd64.tar.gz](https://krustlet.blob.core.windows.net/releases/krustlet-vX.Y.Z-linux-amd64.tar.gz)
+- [krustlet-vX.Y.Z-macos-amd64.tar.gz](https://krustlet.blob.core.windows.net/releases/krustlet-vX.Y.Z-macos-amd64.tar.gz)
+```
+
+Feel free to bring in your own personality into the release notes; it's nice for people to think
+we're not all robots. =3
```

Comment:
Is this supposed to be `:)`?

Reply:
https://www.urbandictionary.com/define.php?term=%3A3

I'll change this to a winky face. It's not a well-known symbol.